config/routes.rb

```diff
@@ -91,7 +91,11 @@
       end
     end
 
-    resources :organizations, path: 'classrooms', only: [:show]
+    resources :organizations, path: 'classrooms', only: [:show] do
+      member do
+        delete '/remove_user/:user_id', to: 'organizations#remove_user', as: 'remove_user'
```

Comment:
Not a huge fan of this route, what do you think about something like this:

```ruby
delete '/users/:user_id', to: 'organizations#remove_user', as: 'remove_user'
```

It's a little more REST friendly. We really don't need the remove in the route since it's a `DELETE` request.